AirCasting/Dashboard/SessionListView.swift

```diff
@@ -13,15 +13,19 @@ import CoreData
 struct SessionsListView: View {
     @FetchRequest<SensorThreshold>(sortDescriptors: [.init(key: "sensorName", ascending: true)]) private var thresholds
     @StateObject private var coreDataHook: CoreDataHook
-    @Binding var isRefreshing: Bool
+    private var isRefreshing: Binding<Bool>?
     @InjectedObject private var featureFlagsViewModel: FeatureFlagsViewModel
     
     private let listCoordinateSpaceName = "listCoordinateSpace"
     private let selectedSection: DashboardSection
 
-    init(selectedSection: DashboardSection, isRefreshing: Binding<Bool>, context: NSManagedObjectContext) {
+    init(selectedSection: DashboardSection, isRefreshing: Binding<Bool>?, context: NSManagedObjectContext) {
         self.selectedSection = selectedSection
-        _isRefreshing = .init(projectedValue: isRefreshing)
+        if let isRefreshing = isRefreshing {
+            self.isRefreshing = .init(projectedValue: isRefreshing)
+        } else {
+            self.isRefreshing = nil
+        }
```

Comment:
So that we could have ` init(selectedSection: DashboardSection, isRefreshing: Binding<Bool>?, context: NSManagedObjectContext) {
        self.selectedSection = selectedSection
        if let isRefreshing = isRefreshing { self.isRefreshing = .init(projectedValue: isRefreshing) }
        _coreDataHook = .init(wrappedValue: .init(context: context))
    }` (2/2)